elements/nuxeo-document-versions/nuxeo-document-create-version.js

```diff
@@ -64,6 +64,23 @@ Polymer({
         font-weight: normal;
       }
 
+      #createButton {
+        display: inline-block;
+        border: none;
+        padding: 0;
+        margin: 0;
+        text-decoration: none;
+        background-color: transparent;
+        color: inherit;
+        font-family: inherit;
+        font-size: inherit;
+        cursor: pointer;
+        text-align: center;
+        transition: background 250ms ease-in-out, transform 150ms ease;
+        -webkit-appearance: none;
+        -moz-appearance: none;
+      }
```

Comment:
Testing it on Chrome, Safari and Firefox I concluded that we don't need all these styles. Please consider the following change:
```suggestion
      #createButton {
        margin: 0;
        padding: 0;
        background-color: transparent;
        border: none;
      }
```

Reply:
Thankyou @semisse, made the changes.